sklearn/linear_model/bayes.py

```diff
@@ -469,9 +469,8 @@ def fit(self, X, y):
         self.scores_ = list()
         coef_old_ = None
 
-        # Iterative procedure of ARDRegression
-        for iter_ in range(self.n_iter):
-            # Compute mu and sigma (using Woodbury matrix identity)
+        # Compute sigma and mu (using Woodbury matrix identity)
+        def update_sigma(X, alpha_, lambda_, keep_lambda, n_samples):
```

Comment:
i am not sure what is the best: keep the function here or make a private method.
@jnothman WDYT?

Reply:
I am fine leaving it here if the object still pickles

Reply:
Is this tested somewhere?